sentry-spring/src/main/java/io/sentry/spring/opentelemetry/SentryOpenTelemetryAgentWithoutAutoInitConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package io.sentry.spring.opentelemetry;
+
+import com.jakewharton.nopen.annotation.Open;
+import io.sentry.Sentry;
+import io.sentry.SentryIntegrationPackageStorage;
+import io.sentry.SentryOptions;
+import io.sentry.opentelemetry.OpenTelemetryUtil;
+import org.jetbrains.annotations.NotNull;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration(proxyBeanMethods = false)
+@Open
+public class SentryOpenTelemetryAgentWithoutAutoInitConfiguration {
+
+  @Bean
+  @ConditionalOnMissingBean(name = "sentryOpenTelemetryOptionsConfiguration")
+  public @NotNull Sentry.OptionsConfiguration<SentryOptions>
+      sentryOpenTelemetryOptionsConfiguration() {
+    return options -> {
+      SentryIntegrationPackageStorage.getInstance()
+          .addIntegration("SpringBoot3OpenTelemetryAgentWithoutAutoInit");
```

Comment:
Rename Integration to reflect, that this is Spring Boot 2
```suggestion
          .addIntegration("SpringBootOpenTelemetryAgentWithoutAutoInit");
```